README.md

```diff
@@ -371,7 +371,7 @@ If this option is disabled, you are responsible for serialization security. You
 - Protocols for features and knowledge graph serialization
 - Continuously improve our serialization infrastructure for any new protocols
 
-## How to Contribute
+## How to Build
 
 Please read the [CONTRIBUTING](CONTRIBUTING.md) guide.
```

Comment:
Do we need to create a `BUILDING.md` in `docs/guides` directory, and move related contents from `CONTRIBUTING.md` to that file?

Reply:
Thank you for your review. If this is definitely the best, I will find time to output the corresponding documents for build and contribute, and then submit a new PR. In the new PR, I will re-change the layout of the README. This may require You can review it carefully when the time comes. This will take some time.

Reply:
For README formatting, I will consider other mature open source projects and learn from their experience.